lib/lita/adapters/slack/rtm_connection.rb

```diff
@@ -31,8 +31,16 @@ def initialize(robot, config, team_data)
           @websocket_url = team_data.websocket_url
           @robot_id = team_data.self.id
 
-          @slack_users = team_data.users
-          @slack_channels = team_data.channels
+          fork do
+            Lita.logger.debug("Inserting #{team_data.users.size} users")
+            UserCreator.create_users(team_data.users, robot, @robot_id)
+            Lita.logger.debug("Done inserting users.")
+
+            Lita.logger.debug("Inserting #{team_data.channels.size} channels")
+            RoomCreator.create_rooms(team_data.channels, robot)
+            Lita.logger.debug("Done inserting channels.")
+          end
+          Process.waitall if ENV['RUBY_ENV'] == 'test'
```

Comment:
Not sure if setting up a `RUBY_ENV`-like variable is the best way to handle this; basically, the tests might finish too fast for the child process to exit, which leads to intermittent test failures on `Lita::Adapters::Slack::RTMConnection#run when the WebSocket is closed from outside shuts down the reactor`